### PR TITLE
Fixed shop api /verify-account route

### PR DESCRIPTION
**Pull request by nevoeiro**
In order to complete the activation of new registered users I propose to change the method of `sylius_shop_api_user_verification` route to [GET] so we are able to access the activation url sent in ShopApiPlugin:Email:verification.html.twig template, when a user registers through shop api.

### CODE REVIEW

File: src/Request/VerifyAccountRequest.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ class VerifyAccountRequest
 
     public function __construct(Request $request)
     {
-        $this->token = $request->request->get('token');
+        $this->token = $request->get('token');
```
Comment (mamazu) [inline src/Request/VerifyAccountRequest.php]: This is inefficient. If you know where the data comes from (in this case the request body) you don't need to search for it everywhere.

Comment (nevoeiro) [reply src/Request/VerifyAccountRequest.php]: The `token` comes from a query parameter. With this approach we support both query or request parameters. The link that is generated is like `http://shop/shop-api/US_WEB/verify-account?token=TLpPvSZOQ6GFVrkU`. So to have it working should be like `$request->query->get('token')`

Comment (mamazu) [reply src/Request/VerifyAccountRequest.php]: Yes, that is what I meant.

**Comment by lchrusciel** [comment]
Thanks, @nevoeiro! :1st_place_medal:

**Pull request merged**